dev.rollup.config.js

```diff
@@ -40,20 +43,21 @@ export default Object.assign({}, umdConfig, {
         dir: tmpDestPath
     }),
     watch: {
-        clearScreen: false
+        clearScreen: false,
     },
     plugins: [
         ...umdConfig.plugins,
         {
             name: 'process-examples',
             async buildStart() {
-                this.addWatchFile(path.resolve('examples'));
+                this.addWatchFile(examplesPath);
                 await tsc(); // Check typescript types
             },
             watchChange(id) {
                 // Rebuild individual example file when changed
-                if (id.indexOf('examples/') < 0) return;
-                buildExample(tmpDestPath, path.relative(__dirname, id));
+                if (id.startsWith(examplesPath)) {
```

Comment:
i wasn't 100% sure how rollup would process and send the id.  i.e. I wasn't sure if the id could be a temp file versus the proper src location or a different root.

But, I tested it and the watch is definitely working and it's just the dev server if anything changes int he future, so this one looks good to me 👍